Blockchain/readme.md

```diff
@@ -17,6 +17,10 @@
 - [What iS A DAO](https://www.forbes.com/sites/cathyhackl/2021/06/01/what-are-daos-and-why-you-should-pay-attention/?sh=343b04067305)
 - [Intro To DAPPS](https://ethereum.org/en/developers/docs/dapps/)
 - [EVM and Non-EVM Chains](https://ethereum.org/en/developers/docs/dapps/)
+- [Metaschool: Free cources](https://metaschool.so/courses)
+- [Lumos Academy: Learning Platform](https://academy.lumoslabs.co/courses)
+- [DeFi Infrastructure - Youtube](https://www.youtube.com/playlist?list=PLE1Vu6ctbqa61FqJmPDnoPzDqrnyILHRm)
+- [Learn by reading Blogs](https://parthmandale.hashnode.dev/series/blockchain-defi)
```

Comment:
These seem to be a complete learning roadmap for blockchain. It would be better to make a separate section at the end of the roadmap called `Additional resources` and link these in there

Reply:
Alright @siddhant-khisty ser, I'm adding `Additional resources for Blockchain Fundamentals` next to it.